packages/core/src/browser/addEventListener.spec.ts

```diff
@@ -33,4 +33,37 @@ describe('addEventListener', () => {
       expect(zoneJsPatchedRemoveEventListener).not.toHaveBeenCalled()
     })
   })
+
+  describe('Untrusted event', () => {
+    it('should be ignored if __ddIsTrusted is absent', () => {
+      const listener = jasmine.createSpy()
+      const eventTarget = document.createElement('div')
+      addEventListener(eventTarget, DOM_EVENT.CLICK, listener)
+
+      const event = createNewEvent(DOM_EVENT.CLICK, { __ddIsTrusted: undefined })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
      const event = createNewEvent(DOM_EVENT.CLICK)
```
or is it needed for some reason?